org.omg.sysml.xtext.ui/plugin.xml_gen

```diff
@@ -330,68 +330,6 @@
 			</command>
 		</menuContribution>
 	</extension>
-	<!-- quickfix marker resolution generator for org.omg.sysml.xtext.SysML -->
```

Comment:
Similar to the KerML extensions, these changes should be copied to the corresponding plugin.xml file.